ee/vellum_cli/ping.py

```diff
@@ -14,6 +17,13 @@ def ping_command():
     try:
         workspace = client.workspaces.workspace_identity()
         organization = client.organizations.organization_identity()
+
+        api_url = os.getenv("VELLUM_API_URL", "https://api.vellum.ai")
+        api_key = os.getenv("VELLUM_API_KEY")
+
+        environment_response = requests.get(f"{api_url}/v1/environments/identity", headers={"X-API-KEY": api_key})
+        environment_response.raise_for_status()
+        environment = environment_response.json()
```

Comment:
call `client.environments.environment_identity()` instead. We will later rebase main to make sure that this doesn't throw a mypy error